app/utils/UserFunctionsWrapper.scala

```diff
@@ -42,7 +42,7 @@ trait UserFunctionsWrapper {
     def toKeyValue(api: APIDefinition, version: ApiVersionDefinition) = {
```

Comment:
Can we rename this as versionDefn to disambiguate and remove the version.version mentions